docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -166,8 +192,12 @@ The panel can be found in `Panels > Add New Panel > tier4_state_rviz_plugin > Au
 
 Now the vehicle should drive along the calculated path!
 
-## 7. Tune parameters for your vehicle & environment
+## 8. Tune parameters for your vehicle & environment
 
 You may need to tune your parameters depending on the domain in which you will operate your vehicle.
 
+Note that there are some kinds of parameter files in Autoware.
+The parameter files you should modify for integration is in your forked Autoware Launch repository cloned in Section. 4.
+You can see more deail about the parameter management in Autoware [here](https://autowarefoundation.github.io/autoware-documentation/pr-171/contributing/coding-guidelines/ros-nodes/parameters/).
```

Comment:
Need to fix typo. Also, the link to the coding guidelines page for parameters should be relative I think.